main.go

```diff
@@ -64,17 +64,24 @@ func main() {
 	usage += "\n"
 
 	// Parse arguments
-	arguments, _ := docopt.Parse(usage, nil, true, "taskcluster", true)
+	arguments, _ := docopt.Parse(usage, nil, true, "TaskCluster version 1.0", true)
```

Comment:
hmm... If we write in capital it's the name and the name of the project is "TaskCluster CLI" or something like that.
the lowercase "taskcluster" was intended to be the name of the binary.

Hmm... are there any GNU guidelines for version string formatting (I can't see any consistency between, less, vim, curl, wget, man, gpg, so I'm guessing no. Maybe do a search through GNU manuals.. Or jsut go with something like `taskcluster (TaskCluster CLI) version %d.%d.%d` with `fmt.Sprintf`

Yes, designing a version message is totally bike-shredding :) Just nice to have fun stuff..